spec/apruve/resources/corporate_account_spec.rb

```diff
@@ -53,6 +54,18 @@
       end
     end
 
+    context 'email with special characters' do
+      let! (:stubs) do
+        faraday_stubs do |stub|
+          stub.get("/api/v4/merchants/#{merchant_uuid}/corporate_accounts?email=#{special_email}") { [200, {}, '{}'] }
+        end
+      end
+      it 'should get a corporate account' do
+        Apruve::CorporateAccount.find(merchant_uuid, special_email)
+        stubs.verify_stubbed_calls
+      end
+    end
+
```

Comment:
This test is verifying that Faraday is being called with an unencoded email address (the value of `special_email`), which is the opposite of what we want to be testing for.  It should be verifying that the URL being used is the encoded version.